packages/web3-providers/src/NextID/proof.ts

```diff
@@ -86,18 +87,19 @@ export class NextIDProofAPI implements NextIDBaseAPI.Proof {
         const cacheKeyOfQueryPersona = getPersonaQueryURL(NextIDPlatform.NextID, personaPublicKey)
         const cacheKeyOfQueryPlatform = getPersonaQueryURL(platform, identity)
         const cacheKeyOfExistedBinding = getExistedBindingQueryURL(platform, identity, personaPublicKey)
-        deleteCache(cacheKeyOfQueryPersona)
-        deleteCache(cacheKeyOfQueryPlatform)
-        deleteCache(cacheKeyOfExistedBinding)
+
+        await staleNextIDCached(cacheKeyOfExistedBinding)
+        await staleNextIDCached(cacheKeyOfQueryPersona)
+        await staleNextIDCached(cacheKeyOfQueryPlatform)
 
         return result
     }
 
     async queryExistedBindingByPersona(personaPublicKey: string, enableCache?: boolean) {
         const url = getPersonaQueryURL(NextIDPlatform.NextID, personaPublicKey)
-        const response = await fetchJSON<NextIDBindings>(url, {}, enableCache)
+        const response = await fetchJSON<NextIDBindings>(url, undefined, fetchSquashed)
```

Comment:
Can I use `fetchJSON` `fetchSquanshed` `fetchR2D2` in the sametime? @guanbinrui

Reply:
Yes, check `packages/web3-providers/src/helpers/fetch.ts`, and example: `packages/mask/background/services/helper/r2d2Fetch.ts`.